e2e/about-how-modal.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test } from '@playwright/test';
+import {
+	ABOUT_HOW_MODAL,
+	ABOUT_HOW_MODAL_OPEN_BUTTON
+} from '../src/frontend/src/lib/constants/test-ids.constant';
+import { ABOUT_MODALS_VIEWPORT_WIDTH } from './utils/constants/e2e.constants';
+import { HomepageLoggedOut } from './utils/pages/homepage.page';
+
+export const ABOUT_HOW_MODAL_VIEWPORT_HEIGHT = 1600;
+
+test('should display about-how modal', async ({ page }) => {
+	const homepageLoggedOut = new HomepageLoggedOut({ page });
+
+	await homepageLoggedOut.takeModalSnapshot({
```

Comment:
since it is already doing the test in it, I would call it something like `testModalSnapshot`, WDYT?

BTW, I was thinking something different with my suggestion: instead of being a function of `HomepageLoggedOut`, it would have been a common test, that takes as input the page too. But you can refactor in another PR if you want, or still this one? WDYT?

Reply:
Agree, `testModalSnapshot` makes more sense in this case, renamed.

Regarding your second point - as soon as we start creating tests for other pages (e.g. `SettingsPage`), we'll need to transform the current `Homepage` class into something even more general, so all future pages can inherit it. The modal functionality will also go into this parent class. If we want to separate Modal from Page completely, it would mean we need to pass not only e.g. `HomepageLoggedOut` instance (so we have access to its custom methods like `waitFor`) but also playwright `Page` object (so we can access DOM-related methods, e.g. `getByTestId`). 

So, I suggest we keep the current structure for now and update it if needed as the e2e tests codebase grows.

Reply:
perfect! agreed!